ActionSheet.android.js

```diff
@@ -161,8 +159,7 @@ export default class ActionSheet extends React.Component {
     BackAndroid.addEventListener('actionSheetHardwareBackPress', this._animateOut);
   }
 
-  @autobind
-  _onSelect(index) {
+  _onSelect = (index) => {
```

Comment:
@ide - how do you feel about this pattern vs `this._onSelect = this._onSelect.bind(this);` in constructor?